_docs/_latest/configure/index.md

```diff
@@ -20,9 +20,9 @@ and Enforcer.
 
 **[Configuring Inventory]({% link _docs/latest/configure/inventory/index.md %})**
 
-Configure Inventory to collect and store information about your GCP resources. Inventory helps you
-understand your resources and take action to conserve resources, reduce cost, and minimize security
-exposure.
+Configure Inventory to collect and store information about your Google Cloud Platform (GCP) resources.
+Inventory helps you understand your resources and take action to conserve resources, reduce cost, and
+minimize security exposure.
```

Comment:
'security risk' might be a better phrase here.

Reply:
Done.